Tests/RedisKitTests/RedisKitTests.swift

```diff
@@ -14,6 +14,126 @@ final class RedisKitTests: XCTestCase {
         }.wait()
     }
 
+    func testCRUD() throws {
+        try self.client.set("hello", to: "world").wait()
+        let get = try self.client.get("hello", as: String.self).wait()
+        XCTAssertEqual(get, "world")
+        let _ = try self.client.delete(["hello"]).wait()
```

Comment:
It would be nice to have a `String...` overload for this so you could do:
```swift
self.client.delete("hello")
// or 
self.client.delete("hello", "foo")
```

Reply:
where would you like to put that?

Reply:
Yeah I figured it was part of redistack :)

Reply:
Added several overloads to RediStack: https://gitlab.com/Mordil/swift-redi-stack/merge_requests/87